cli/init/index.js

```diff
@@ -1,50 +1,82 @@
 #!/usr/bin/env node
 
 (async () => {
-  const { constants, promises } = require('fs')
-  const { access, mkdir, readFile, writeFile } = promises
+  const { promises, existsSync } = require('fs')
+  const { mkdir, readFile, writeFile } = promises
   const fileOptions = { encoding: 'utf8' }
   const dir = "src"
   const beagleContent = await readFile(__dirname + '/boilerplate/beagle-service.ts')
   const appContent = await readFile(__dirname + '/boilerplate/app.tsx')
+  const appTsx = 'app.tsx'
+  const beagleServicePath = 'beagle/beagle-service.ts'
   const readline = require('readline')
   const rl = readline.createInterface({
     input: process.stdin,
     output: process.stdout,
   })
 
   await mkdir(`${dir}/beagle/`, { recursive: true });
-  await writeFile(`${dir}/beagle/beagle-service.ts`, beagleContent, fileOptions)
 
-  const createAppTsx = async (path, content) => {
+  const beagleServiceExists = async (path) => {
+    const exists = await existsSync(`${path}/${beagleServicePath}`)
+    return exists
```

Comment:
```suggestion
    return await existsSync(`${path}/${beagleServicePath}`)
```
You can return the result directly without a variable